src/main/java/org/osc/controller/nsc/restserver/api/InspectionHookApis.java

```diff
@@ -0,0 +1,163 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.controller.nsc.restserver.api;
+
+import static java.util.Collections.singletonMap;
+import static org.osgi.service.jdbc.DataSourceFactory.*;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import javax.persistence.EntityManager;
+import javax.sql.DataSource;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.log4j.Logger;
+import org.osc.controller.nsc.api.SampleSdnRedirectionApi;
+import org.osc.controller.nsc.entities.InspectionHookEntity;
+import org.osc.controller.nsc.utils.RedirectionApiUtils;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.jdbc.DataSourceFactory;
+import org.osgi.service.jpa.EntityManagerFactoryBuilder;
+import org.osgi.service.transaction.control.TransactionControl;
+import org.osgi.service.transaction.control.jpa.JPAEntityManagerProviderFactory;
+
+@Component(service = InspectionHookApis.class)
+@Path("/inspectionHook")
+@Consumes({ MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML })
+@Produces({ MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML })
+public class InspectionHookApis {
+    private static Logger LOG = Logger.getLogger(InspectionPortApis.class);
+    private EntityManager em;
+    private RedirectionApiUtils utils;
+    private SampleSdnRedirectionApi sdnApi;
+    @Reference(target = "(osgi.local.enabled=true)")
```

Comment:
add line spaces in between the declaration blocks.

I see 3 blocks , the log, the private variable and private @reference variables.

also add lines spaces between each of the annotated variables similiar to the sytle in OSC. I will try to enforce this via checkstyle but for now please add line space like below

```
@Reference(target = "(osgi.unit.name=nsc-mgr)")
private EntityManagerFactoryBuilder builder;

@Reference(target = "(osgi.jdbc.driver.class=org.h2.Driver)")
private DataSourceFactory jdbcFactory;
```